src/arch/riscv/kernel/vspace.c

```diff
@@ -105,14 +105,13 @@ BOOT_CODE void map_kernel_frame(paddr_t paddr, pptr_t vaddr, vm_rights_t vm_righ
 
 BOOT_CODE VISIBLE void map_kernel_window(void)
 {
-    /* mapping of KERNEL_ELF_BASE (virtual address) to kernel's
-     * KERNEL_ELF_PHYS_BASE  */
+    /* mapping of physical memory, kernel ELF image and devices. */
     assert(CONFIG_PT_LEVELS > 1 && CONFIG_PT_LEVELS <= 4);
 
     /* kernel window starts at PPTR_BASE */
     word_t pptr = PPTR_BASE;
 
-    /* first we map in memory from PADDR_BASE */
+    /* first we map physical memory from PADDR_BASE */
```

Comment:
"map in" is a correct idiomatic expression, I don't see any need to change it. We could add `physical` to make it extra clear that this is not about devices, but `PADDR_BASE` does already imply that.